src/thunderbots/software/ai/passing/evaluation.cpp

```diff
@@ -34,25 +35,31 @@ double AI::Passing::ratePass(const World& world, const AI::Passing::Pass& pass,
         in_region_quality = rectangleSigmoid(*target_region, pass.receiverPoint(), 0.1);
     }
 
-    double pass_quality = static_pass_quality * friendly_pass_rating * enemy_pass_rating *
-                          shoot_pass_rating * in_region_quality;
-
-    // Strict requirement that the pass occurs at a minimum time in the future
+    // Place strict limits on pass start time
     double min_pass_time_offset =
         Util::DynamicParameters::AI::Passing::min_time_offset_for_pass_seconds.value();
+    double max_pass_time_offset =
+            Util::DynamicParameters::AI::Passing::max_time_offset_for_pass_seconds.value();
+    double pass_time_offset_quality = 1;
     // TODO (Issue #423): We should use the timestamp from the world instead of the ball
-    pass_quality *= sigmoid(
+    pass_time_offset_quality *= sigmoid(
```

Comment:
Avoiding the `*=` operator would be a bit cleaner

Reply:
Good point, changed.